### ISSUE TITLE
Adjust open App Store page with Android app installed in Chrome Browser

### ISSUE THREAD

**Issue by leticiarod** [adjust/android_sdk#304]
Hi! I'm having problems to open an android app when using an adjust URL on Chrome browser with the deep-link parameter in it. 

My encoded URL looks like this:

https://app.adjust.com/xxxxxx?deep_link=example%3A%2F%2FfisrtPath%2FsecondPath

The problem is that when i directly paste the above url on the chrome browser the app doesn't open and instead the app store page opens with the android app already installed.

Does anyone know which could be the problem here? There is no information about this on the internet whatsoever or at least i didn't find any.

Thank you in advance!

**Comment by uerceg** [comment]
Hi @leticiarod 

I don't think that direct copy/paste of the URL into URL bar and trying to load the page works. The link is supposed to be clicked, so if you can maybe try to put this URL on some webpage, load it in Chrome and try to click onto it.

Let us know how did the test go.

Cheers

**Comment by leticiarod** [comment]
Yes, i also did that and doesn't work. 
I did like a local web page in which i put this html, like as follow:

<a href="https://app.adjust.com/xxxxxx?deep_link=example%3A%2F%2FfistPath%2FsecondPath">Test</a>

In chrome when i click on "Test" adjust redirects to the app store too.

Thank you!

**Comment by uerceg** [comment]
🤔 Lemme just make sure that we think of same thing - when you say App Store, to you think of iOS App Store or Google Play Store?

**Comment by leticiarod** [comment]
iOS App Store 

redirects to this link: https://itunes.apple.com/app/idxxxxxxxxx?mt=z

**Comment by uerceg** [comment]
👍 

Are you sure that token you are using in your URL (`xxxxxx`) is tracker token for your Android app and not maybe some iOS app in the dashboard?

**Comment by uerceg** [comment]
Do you see this happening all the time you try to click on your URL? Does it happen in both scenarios - when your app is not installed on test device and when it is installed or in just one of them?

**Comment by leticiarod** [comment]
In the emulator occurs in both scenarios when the app is installed and when it isn't.

**Comment by uerceg** [comment]
Would it be possible for you to try this from an actual device?

**Comment by leticiarod** [comment]
Yes, also tried with a device, what I did was send this URL (
https://app.adjust.com/xxxxxx?deep_link=example%3A%2F%2FfirstPath%2FsecondPath) by email and happens the same thing, I get the same error.

**Comment by uerceg** [comment]
Okay. Can you please send email to support@adjust.com and reference this question into it and provide us with full URL including your tracker token and then I’ll run the test as well to see what’s up?

**Comment by leticiarod** [comment]
done

**Comment by uerceg** [comment]
Thanks for the email. 

Hm. I tried the link from Chrome browser and once app was not installed, it guided me to the Play Store and once I have downloaded the app and clicked the link, app got opened.

Really weird.

In my test HTML page, I added your link like this:

```html
<p><a href="https://app.adjust.com/[ABC123]?deep_link=[SCHEME]%3A%2F%2FfirstPath%2FsecondPath">Adjust Insights Github Issue</a></p>
```

**Comment by leticiarod** [comment]
mmm And what happens when the app is installed? The link opens the app too?  

I copied/pasted that same URL and keeps given me that same error of "This site can't be reached"

Just in case, the straight parentheses are not necessary, right?

**Comment by uerceg** [comment]
Yeah, right, this was just me obfuscating your info. Really really weird, I tried with several devices and it worked. Not sure if it makes any difference, I am running my tests with dummy page made on neocities.org website. 

I will check with our backend team is there anything that can cause this.

**Comment by leticiarod** [comment]
Ok, thank you !! I wait for your answer then! In the meantime I'll keep testing to try to find an answer!

**Comment by leticiarod** [comment]
I just realized that your test is doing it with the app in production, im testing it with changes that are not in prod. : / yes yes, it would be nice to check with the backend team to get at least a clue of what's happening, to know under what circumstances Adjust gives that error ..

Also, could the problem have something to do with the APP ID? I mean, that the APP ID have to correspond with SCHEME of the deeplink?

Thank you !!!

**Comment by leticiarod** [comment]
I think I found what my problem was. I found that my dashboard in Adjust only had the App that is in production.

I added another app, the app that is in testing stage the one that is modifying (this one is not in play store, clearly) but i keep receiving that error message:

This site can not be reached

>
> The webpage at intent: // example / firstPath? Adjust_reftag = cMF27c5ztLdCn # Intent; scheme = myApp; package = com.myApp; end might be temporarily down or it may have moved permanently to a new web address.
> ERR_UNKNOWN_URL_SCHEME

I understand that the package name corresponds to the APP ID in adjust, right?
So in this new app I added to the dashboard, I changed the APP ID, which is APP ID = com.myApp.staging.

So, if you take a look at the above message, you will see that package name = "com.MyApp" that is, the package name corresponding to the APP in prod not the one I'm testing, although I've already added the app in test stage to the dashboard with it corresponding APP ID (or package name).

I already set the new APP Token in my code, the one corresponding to the App added to the dashboard in adjust, and also did a Clean and Build as said in the "Getting started" section in Adjust Docs.

Do you know if there's something else I need to do in order to "refresh" where Adjust is redirecting the URL?

I do not know if I explain myself correctly .. let me know if something does not let clear.

Thank you again !!

**Comment by leticiarod** [comment]
Sorry! I was passing in a wrong tracker ID!

After start passing a correct Tracker ID the error doesn't show anymore.

Thank you!
